website/docs/r/project.html.markdown

```diff
@@ -15,9 +15,12 @@ description: |-
 ## Example Usage
 
 ```terraform
+data "mongodbatlas_roles_org_id" "test" {
+}
```

Comment:
@martinstibbe should this data block be empty?

Reply:
No parameters just using API key to connect terraform to API gets org_id value